xarray/tests/test_coding_times.py

```diff
@@ -1438,10 +1451,8 @@ def test_roundtrip_float_times(fill_value, times, units, encoded_values) -> None
         "days since 1700-01-01",
         np.dtype("int32"),
     ),
-    "mixed-cftime-pandas-encoding-with-prescribed-units-and-dtype": (
-        "250YS",
-        "days since 1700-01-01",
-        np.dtype("int32"),
+    "mixed-cftime-pandas-encoding-with-prescribed-units-and-dtype": pytest.param(
+        "250YS", "days since 1700-01-01", np.dtype("int32"), marks=requires_cftime
```

Comment:
Not related to this PR, but I noticed when testing in an environment without cftime that this test was not skipped.  I guess we don't have a CI environment that has dask, but not cftime.  I figured I would take the opportunity to clean this up.